deploy/operator.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       serviceAccountName: pravega-operator
       containers:
       - name: pravega-operator
-        image: pbelgundi/pravega-operator:m67
+        image: pravega/pravega-operator:0.5.0-rc1
```

Comment:
Use image tag "latest" as that makes sure we don't need to update this file every time we do a new release.

Reply:
I have changed tag to latest